src/redux/auth/auth-selectors.js

```diff
@@ -0,0 +1,6 @@
+export const getIsAuth = state => {
+  return state.auth.isAuth;
```

Comment:
Not really that long to be exported to a separate file tbh. Also can be a one-liner. You would usually want to move selectors if you are using more libs like reselect or similar.